pkg/registry/apis/apis.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/grafana/grafana/pkg/registry"
 	"github.com/grafana/grafana/pkg/registry/apis/example"
+	"github.com/grafana/grafana/pkg/registry/apis/folders"
```

Comment:
we should set a standard for the group names, right now we have playlist (singular) and folders (plural)...

Reply:
🤔 any preference?  I flip back and forth, and in general looks like there is not a fixed convention.  If it is a single resource, the plural will be in the URL.

Reply:
plural feels natural for many, but not all -- i don't have any strong convictions here :)

Reply:
for this case I think plurals make more sense, playlists.grafana.app feels more natural than folder.grafana.app

Reply:
I did some testing with the unified storage and I think that the [FolderGroupName](https://github.com/grafana/grafana/blob/52dc42545826b4bf51a4e1c28e11c30a134db9b2/pkg/services/store/entity/models.go#L10) should also change to plural (otherwise the folders are not stored in the [entity_folder](https://github.com/grafana/grafana/blob/52dc42545826b4bf51a4e1c28e11c30a134db9b2/pkg/services/store/entity/sqlstash/sql_storage_server.go#L474))

Reply:
I updated the keys here, and got it working enough that [updateFolderTree](https://github.com/grafana/grafana/blob/b5e19cf35fdbe6f8c95593a00e6ea903b06c9258/pkg/services/store/entity/sqlstash/folder_support.go#L37) is called and does. not crash, but clearly not working yet!

<img width="1049" alt="image" src="https://github.com/grafana/grafana/assets/705951/4a6d8d08-8d01-4c14-a076-cbd39eeb1f3d">

<img width="1000" alt="image" src="https://github.com/grafana/grafana/assets/705951/fed284d1-6e3d-4446-9d81-7331c4bfa41f">

I'll take a look at that after this is merged... worried about growing the scope too much!